vnext/local-cli/runWindows/utils/autolink.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+// @ts-check
+
+const execSync = require('child_process').execSync;
+const path = require('path');
+const fs = require('fs');
+
+function updateAutoLink(verbose) {
+  const execString =
+    'react-native config';
+  let output;
+  try {
+    console.log('Running react-natice config...');
+    output = execSync(execString).toString();
+    if (verbose)
+    {
+      console.log(output);
+    }
+
+    const config = JSON.parse(output);
+    const windowsPlatformConfig = config['project']['windows'];
+    const cppProjFile= windowsPlatformConfig?windowsPlatformConfig.cppProjFile:null;
+    if (cppProjFile == null){
+      console.log('AutoLink currently is only supported on cpp/winrt main project.');
+      return;
+    }
+    console.log('cppProjFile:' + cppProjFile);
+
+    const sourceDir = config['project']['windows'].sourceDir;
+    console.log('sourceDir:' + sourceDir);
+
+    const projectName = config['project']['windows'].projectName;
+    console.log('projectName:' + projectName);
+    
+    //#1. update nativeModules.g.h
+    const generatedHeader =  path.join(sourceDir, projectName, 'nativeModules.g.h');
+    if (!fs.existsSync(generatedHeader))
+    {
+      console.log('autoLink can not locate generated header file:' + generatedHeader);
+      return;
+    }
+
+    var generatedIncludes = '// NativeModules.g.h -- contents generated by "react-native run-windows"';
```

Comment:
nit: We shouldn't use `var` in modern JS since it has surpising semantics around function-local scope. Only use `let` or `const`